ci/compress-tasks.ps1

```diff
@@ -10,6 +10,17 @@ param(
 
 $ErrorActionPreference = 'Stop'
 Add-Type -Assembly 'System.IO.Compression.FileSystem'
+Add-Type -AssemblyName 'System.Text.Encoding'
+
+class PathSeparatorEncoder : System.Text.UTF8Encoding {
+    PathSeparatorEncoder() : base($true) { }
+
+    [byte[]] GetBytes([string] $s)
+    {
+        $s = $s.Replace("\", "/");
```

Comment:
I am not a PS expert. But does the "\" need to be escaped. I.e. "\\" ??

Reply:
I am also not :) I tested it twice, once with `"\"` and once with `"\\"` and it looks like it's working the way I have it.